README.md

```diff
@@ -27,6 +27,8 @@ Configuration of the adapter is done via environment variables at startup.
 | `SCANNER_API_SERVER_ADDR`          | `:8080` | Binding address for the API HTTP server. |
 | `SCANNER_API_SERVER_TLS_CERTIFICATE` | | The absolute path to the x509 certificate file. |
 | `SCANNER_API_SERVER_TLS_KEY`         | | The absolute path to the x509 private key file. |
+| `SCANNER_HTTP_TLS_INSECURE_SKIP_VERIFY` | `false` | Controls whether an HTTP client verifies the server's certificate chain and host name. |
+| `SCANNER_HTTP_TLS_CLIENTCAS` | | An array of absolute paths to x509 CA files that will be added to host's root CA set. |
```

Comment:
`HTTP_TLS` in `SCANNER_HTTP_TLS_xxx` is not very clearly for http does not need certificates.

These settings are used to communicate with harbor, maybe `SCANNER_HARBOR_TLS_xxx` or `SCANNER_TLS_xxx` would be better?

Reply:
I found that the certificates are also used by clair client, so maybe `SCANNER_TLS_xxx` would be better?

Reply:
Indeed, the TLS config is used by both Docker registry and Clair HTTP clients. I changed the envs prefix to `SCANNER_TLS_xxx`.